.github/workflows/pilotWrapper.yml

```diff
@@ -0,0 +1,23 @@
+name: pilot wrapper
+
+on: [push, pull_request]
+
+jobs:
+  PilotWrapper:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: False
+      matrix:
+        python:
+          - 2.7.13
+          - 3.6.8
```

Comment:
Is there a reason to not test newer versions as well?
```suggestion
          - 3.6.8
          - 3.7
          - 3.8
```

Reply:
I tested 3.6.8 because this is the version that comes within CentOS8. I don't see a reason for testing higher than that.

Reply:
Which, BTW, now that the environment is properly activated, fails! 

I'll check next year...